### PR TITLE
bug: Need accessibility improvements

### PR DESCRIPTION
**Pull request by senali-d**
Accessibility improvements - FAQ

Issue: bug: need accessibility improvements #123

### CODE REVIEW

File: pages/faq/components/FAQsection.tsx [hunk 1]
```diff
@@ -48,21 +49,25 @@ export const FAQsection: FC<IFAQsection> = ({
...
-											{faq}
-										</Typography>
-									</AccordionSummary>
-									<AccordionDetails>
-										<Typography variant="body1">{desc}</Typography>
-									</AccordionDetails>
-								</Accordion>
-							</Box>
-						))}
+						<ul className={styles.list}>
```
Comment (ashuvssut) [inline pages/faq/components/FAQsection.tsx:52]: No need to create extra `ul` elemenet.

You can create the `ul` element by editing Line 51
`<List component="div" disablePadding>` ---> `<List component="ul" disablePadding>` (or maybe simply `<List  disablePadding>`)

**Review by ashuvssut — CHANGES_REQUESTED** [review]

**Comment by ashuvssut** [comment]
You still havent all the fixed the accessibility issues (errors)
image

Please read the issue's description carefully. You will know what need to be done

Refer this comment on PR https://github.com/WalkingPal/walking-pal-web/pull/127#issuecomment-1273714876

**Comment by senali-d** [comment]
@ashuvssut 
- I skipped low contrast issue because you mention in the comment https://github.com/WalkingPal/walking-pal-web/pull/127#issuecomment-1273716663

- Missing form label issue is out of control. Because it is came with MUI.
  Multiline `TextField` generate two `textarea` element but only one element they assign `id`. There is no `id` for the hidden `textarea` element.

textarea

**Comment by ashuvssut** [comment]
If have fixed some of those webaim Alerts (warnings) (ref 965c7e6).

**Comment by ashuvssut** [comment]
@senali-d there are two more pages that need accessibility improvements: Register & 404 page. Your can take those up if you want to work on them.

**Comment by senali-d** [comment]
> @senali-d there are two more pages that need accessibility improvements: Register & 404 page. Your can take those up if you want to work on them.

Yup sure. I hope to work on 404 page.

**Pull request merged**